### ISSUE TITLE
Show 1/3, 2/3 instead of TBC

### ISSUE THREAD

**Issue by cosmo0** [benbarden/switchscores#39]
Allow users to set their own category: I'm not a fan of puzzle games but I'm willing to play more shmups for instance.

Link to game page from list of games and display the category

**Comment by cosmo0** [comment]
New idea: in the "unranked" lists, when the user is a reviewer, display the number of reviews like "1/3" instead of "TBC" (what does it mean?) and display a color when there are 2 reviews (like purple or whatever: "one more to go and it's ranked !")

**Comment by benbarden** [comment]
Which unranked lists do you mean - can you post a link?

**Comment by cosmo0** [comment]
Something like that:

![image](https://user-images.githubusercontent.com/3973850/105323691-b437a880-5bca-11eb-9971-d97e8f0efc74.png)

**Comment by benbarden** [comment]
I've moved the first idea to #41 as these are different things.

I think this is a good idea. "TBC" means to be confirmed, but even knowing that's what it means, it's not obvious it is relating to the fact it doesn't have 3 reviews.

I've got a version of this working - I've just put it live. Have a look around and see if you notice any issues.

I didn't think it needed the additional colour (the purple you've used for 2/3). I've added the word "reviews" underneath so it's clear what it is. It isn't working on all pages yet, but it should use the TBC where I've not set it up yet. For example, the layout I've used on https://www.switchscores.com/games/by-date/2021-01 meant it was quite difficult to get the "reviews" word to appear in the right place. So that page will stay as it is for now.

I've also taken the opportunity to tidy up a few of the layout files while I'm at it:

- The scores used to appear too low on the page when browsing on mobile. That's now fixed;
- The Games on sale page had an odd layout, as did the Top Rated pages. Those are now more consistent with the rest of the site.

In the future, most of the layouts can now be merged. It also means it should be easier to do #7 .
